_component_design/cards.md

```diff
@@ -17,56 +19,28 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-
-### When to use
-
-Cards allow the user to more visually distinguish between sets of information than the Lists pattern. By providing the user with a familiar metaphor – ie: we use cards in the real world – it comes with sets of expectations that make them easy to use. Use cards to present minimal high level information that allows the user to dig deeper. Do not use cards in place of hyper-robust data tables, also be wary of using cards with a large quantity of data entries.
-
-### Best practice
-
-Cards should be uniform in size. Don’t overcrowd your card. It is not a Dashboard Card, which is a pattern that will be developed in the future. It is not meant to hold a wealth of information, but high level views to dig deeper into. Cards are not always the right solution to displaying information. If the information would be better presented in a list or table, do so, or if we have reason to believe Card Grids will appeal to some users, present both view options.
-
-{% endcolumn %}
-
-{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% figure [class:"image bg-light border"] %}
- ![]({{site.url}}/assets/images/components/content-areas/cards/cards-hero.png){:width="600px"}
-{% endfigure %}
-{% endcolumn %}
-
-</div>
-
-</section>
-
-## Overview
-
-<section class="static-section" markdown="1">
-
-<div class="hxRow"  markdown="1">
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
 
-### Default layout
+## When to use
 
+Use cards when you want to a visually distinct arrangement of tabular data, especially when it is helpful to provide summary information about that data object.
```

Comment:
s/`when it is helpful to provide summary information about that data object`/`when an informational summary of different data objects is helpful for the user`